webapp/data_structures/FROMWS_c.js

```diff
@@ -0,0 +1,30 @@
+function FROMWS_c () {
+
+    FROMWS_c.prototype.define = function FROMWS_c() {
+        this.varnam = "V";
+        this.Method = 1;
+        this.ZC = 1;
+        this.OutEnd = 0;
+        this.str2code = -31; //this is value which we get from _str2code(varnam) function.
```

Comment:
```suggestion
        this.str2code = _str2code("V");
```

Please check if we can do this.